### ISSUE TITLE
QAT with CUDA

### ISSUE THREAD

**Issue by dtcrong** [pytorch/pytorch#28653]
Try post train with my own network and dataset but accuracy is not good enough. 
Try QAT with too slow. 
Any way to quickly run QAT on NVIDIA GPU?

cc @jerryzh168 @jianyuh @dzhulgakov @raghuramank100 @jamesr66a

**Comment by z-a-f** [comment]
CUDA quantization is not yet supported.

**Comment by z-a-f** [comment]
QAT supports CUDA through (FakeQuantize)[https://pytorch.org/docs/stable/quantization.html#torch.quantization.FakeQuantize]. You can follow the (tutorial)[https://pytorch.org/tutorials/advanced/static_quantization_tutorial.html#quantization-aware-training], but you will need to adapt it to use CUDA.

**Comment by z-a-f** [comment]
@dtcrong Closing this. If you have any questions about quantization, you can direct it to the [Forum](https://discuss.pytorch.org/c/quantization)
